src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -239,6 +289,47 @@ export const NewProposal = ({ navigation, data, query }) => {
               </ScrollView>
             </>
           )}
+
+          {item.type === ITEM_TYPES.PICKER && (
+            <Controller
+              key={item.name}
+              name={item.name}
+              control={control}
+              render={({ onChange, value }) => {
+                useEffect(() => {
+                  if (!value) {
+                    onChange(imageUri);
+                  }
+                }, [imageUri]);
```

Comment:
this needs to be placed in the main component and not inside the render prop here. this seems bad practice to me.